src/components/modals/HelpModal.js

```diff
@@ -75,6 +75,22 @@ export class HelpModalComponent extends React.Component<Props & ThemeProps> {
     const helpModalTitle = sprintf(s.strings.help_modal_title_thanks, config.appName)
     const helpSiteMoreInfoText = sprintf(s.strings.help_site_more_info_text, config.appName)
 
+    const handleEdgeSitePress = () => {
+      if (Platform.OS === 'android') {
+        Linking.canOpenURL(HELP_URIS.site).then(supported => {
+          if (supported) {
+            Linking.openURL(HELP_URIS.site).catch(err => {
+              console.log(err)
+            })
+          } else {
+            console.log("Don't know how to open URI: " + HELP_URIS.site)
+          }
+        })
+      } else {
+        showWebViewModal(HELP_URIS.site, helpSiteMoreInfoText)
+      }
+    }
+
```

Comment:
Since the HelpModalComponent is a class component, you should add this function as a class function (like the 'handleClose' above)